socketcan_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(socketcan_bridge)
+
+find_package(catkin REQUIRED COMPONENTS
+  socketcan_interface
+  roscpp
+  can_msgs
+  roslint
+)
+
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES socketcan_to_topic topic_to_socketcan socketcan_interface roscpp can_msgs
+  CATKIN_DEPENDS 
+  DEPENDS Boost
+)
+
+include_directories(include ${Boost_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS})
+
+
+add_library(socketcan_to_topic src/socketcan_to_topic.cpp)
+target_link_libraries(socketcan_to_topic ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+add_dependencies(socketcan_to_topic can_msgs_generate_messages_cpp)
```

Comment:
The recommended way is to depend on `${catkin_EXPORTED_TARGETS}`, see http://docs.ros.org/indigo/api/catkin/html/howto/format2/cpp_msg_dependencies.html